docs/misc/deprecated.md

```diff
@@ -0,0 +1,64 @@
+<!--[metadata]>
++++
+title = "Docker Deprecated Features"
+description = "Deprecrated Features."
+keywords = ["docker, documentation, about, technology, deprecate"]
+[menu.main]
+parent = "mn_use_docker"
++++
+<![end-metadata]-->
+
+# Deprecated Features
+
+The following list of features are deprecated.
+
+### Old Command Line Options
+**Target For Removal In Release: v1.10**
+
+The following single-dash (`-opt`) variant of certain command line options
+are deprecrated:
+
+    docker attach -nostdin
+    docker attach -sig-proxy
+    docker build -no-cache
+    docker build -rm
+    docker commit -author
+    docker commit -run
+    docker events -since
+    docker history -notrunc
+    docker images -notrunc
+    docker inspect -format
+    docker ps -beforeId
+    docker ps -notrunc
+    docker ps -sinceId
+    docker rm -link
+    docker run -cidfile
+    docker run -cpuset
+    docker run -dns
+    docker run -entrypoint
+    docker run -expose
+    docker run -link
+    docker run -lxc-conf
+    docker run -n
+    docker run -privileged
+    docker run -volumes-from
+    docker search -notrunc
+    docker search -stars
+    docker search -t
+    docker search -trusted
+    docker tag -force
+
+Note that the double-dash (`--opt`) variant of any of these options are still
+supported.
+
+Additionally, the following options are also deprecated:
+
+    docker run --networking
+    docker ps --since-id
+    docker ps --before-id
+    docker search --trusted
+
+**Replacement:**
+
+Many of the deprecated options have a double-dash variant that should be
+used instead. There is no replacement option available for those that do not.
```

Comment:
Can this document also list or link to the API changes - this might be something that could be automated, but we've surprised users a number of times, so over-documentation is probably worth it.